pandas/tests/groupby/aggregate/test_numba.py

```diff
@@ -134,25 +134,129 @@ def func_1(values, index):
 
 
 @td.skip_if_no("numba")
+# TODO: Add test to check that UDF is still jitted by numba
 @pytest.mark.parametrize(
-    "agg_func",
+    "agg_kwargs",
     [
-        ["min", "max"],
-        "min",
-        {"B": ["min", "max"], "C": "sum"},
-        NamedAgg(column="B", aggfunc="min"),
+        {"func": ["min", "max"]},
+        {"func": "min"},
+        {"func": {1: ["min", "max"], 2: "sum"}},
+        {"bmin": NamedAgg(column=1, aggfunc="min")},
     ],
 )
-def test_multifunc_notimplimented(agg_func):
+def test_multifunc_numba_vs_cython_frame(agg_kwargs):
     data = DataFrame(
-        {0: ["a", "a", "b", "b", "a"], 1: [1.0, 2.0, 3.0, 4.0, 5.0]}, columns=[0, 1]
+        {
+            0: ["a", "a", "b", "b", "a"],
+            1: [1.0, 2.0, 3.0, 4.0, 5.0],
+            2: [1, 2, 3, 4, 5],
+        },
+        columns=[0, 1, 2],
+    )
+    grouped = data.groupby(0)
+    result = grouped.agg(**agg_kwargs, engine="numba")
+    expected = grouped.agg(**agg_kwargs, engine="cython")
+    # check_dtype can be removed if GH 44952 is addressed
+    tm.assert_frame_equal(result, expected, check_dtype=False)
+
+
+@td.skip_if_no("numba")
+@pytest.mark.parametrize(
+    "agg_kwargs,expected_func",
+    [
+        ({"func": lambda values, index: values.sum()}, "sum"),
+        # FIXME
+        pytest.param(
+            {
+                "func": [
+                    lambda values, index: values.sum(),
+                    lambda values, index: values.min(),
+                ]
+            },
+            ["sum", "min"],
+            marks=pytest.mark.xfail(
+                reason="This doesn't work yet! Fails in nopython pipeline!"
+            ),
```

Comment:
Is this failing because it's multiple lambdas, or will it fail on any multiple callables?

Reply:
This was the comment in the OP.